pxr/usd/usdGeom/wrapCamera.cpp

```diff
@@ -129,6 +129,34 @@ _CreateExposureAttr(UsdGeomCamera &self,
     return self.CreateExposureAttr(
         UsdPythonToSdfType(defaultVal, SdfValueTypeNames->Float), writeSparsely);
 }
+        
+static UsdAttribute
+_CreateExposureIsoAttr(UsdGeomCamera &self,
+                                      object defaultVal, bool writeSparsely) {
```

Comment:
Is this spacing offset intentional?

Reply:
Not sure - but every `_CreateXXX` function in that file has the same spacing.  Should probably fix all or none.